optimism.tokenlist.json

```diff
@@ -268,6 +268,14 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 10,
+      "address": "0x65559aA14915a70190438eF90104769e5E890A00",
+      "name": "Ethereum Name Service",
+      "symbol": "ENS",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/ENS.png",
```

Comment:
You need to add the bridge to use for token here, for the standard bridge it is:
```
"extensions": {
        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
      }
```